sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/Utils.java

```diff
@@ -0,0 +1,54 @@
+package com.azure.data.tables.implementation;
+
+import com.azure.data.tables.implementation.models.TableServiceErrorOdataError;
+import com.azure.data.tables.implementation.models.TableServiceErrorOdataErrorMessage;
+import com.azure.data.tables.models.TableServiceError;
+import com.azure.data.tables.models.TableServiceErrorException;
+
+public class Utils {
```

Comment:
Add class level javadoc. Also update the name to `TableUtils` as we already have too many `Utils` class.